### PR TITLE
Updates kmap.html

### LINKED ISSUE #150: Inconsistent Font and Button Text
## What is the issue

The text in the bottom right corner is different e.g. "CircuitVerse on Github" here 
https://learn.circuitverse.org/docs/kmapi.html 
image 
and "Edit on Github" here 
https://learn.circuitverse.org/docs/bool.html
image

The fonts change from from https://learn.circuitverse.org/docs/bool.html
image

to https://learn.circuitverse.org/docs/bool2.html
image

## How to resolve
Change so they are consistent over different pages - not sure which fonts and button text it should be changed to though.

### PR DESCRIPTION
**Pull request by ShubhankarKG**
Fixes #150 
1. Edited text to "Edit on Github" and rectified links
2. Added dark mode featurette. The script didn't render properly.
3. Added support for jquery in kmap.html

## ✅️ By submitting this PR, I have verified the following

* [x] Checked to see if a similar PR has already been opened 🤔️
* [x] Reviewed the contributing guidelines 🔍️

### CODE REVIEW

**Comment by ShubhankarKG** [comment]
Please review @YashKumarVerma

File: _layouts/kmap.html [hunk 1]
```diff
@@ -2,110 +2,136 @@
 
 <html lang="en">
 {% include head.html %}
+<script src="../assets/js/module.js"></script>
+<script src="../assets/js/vendor/jquery-3.4.1.min.js"></script>
+<script src="../assets/js/shareable-links.js"></script>
```
Comment (YashKumarVerma) [inline _layouts/kmap.html:8]: https://github.com/CircuitVerse/Interactive-Book/blob/master/_layouts/default.html  

do we need to include all these? they're already in the default layout

Comment (ShubhankarKG) [reply _layouts/kmap.html:8]: kmap.html is an independent layout. So, it doesn't use the default.html scope variables. I've tried without using them, but jquery doesn't seem to work. Hence, added.

Comment (ShubhankarKG) [reply _layouts/kmap.html:8]: Without these, the dark mode won't work.

Comment (YashKumarVerma) [reply _layouts/kmap.html:8]: cool, just noticed it.

**Comment by ShubhankarKG** [comment]
So, I did a quick lookup and realized that the kmap.html and default.html are the same now. So they can be removed. Also, the four other pages,namely __about.html__, __home.html__, __page.html__ and __post.html__ have the same content. They can be converted into one page as well. Your thoughts ?

**Comment by ShubhankarKG** [comment]
Closing this as the issue is fixed.

**Pull request closed without merging**